web_poet/__main__.py

```diff
@@ -0,0 +1,34 @@
+import argparse
+from typing import Callable
+
+import tabulate
+
+from web_poet.overrides import find_page_object_overrides
+
+
+def qualified_name(cls: Callable) -> str:
+    return f"{cls.__module__}.{cls.__name__}"
+
+
+def main(args=None):
```

Comment:
Very neat! 👏 Thanks for adding this!

We could also add this line to **setup.py**: `entry_points={'console_scripts': ['web_poet = web_poet.__main__:main']}`

So instead of:

```
$ python -m web_poet tests.po_lib
```

We could do something like this directly:

```
$ web_poet tests.po_lib
```

Reply:
Handled this on https://github.com/scrapinghub/web-poet/pull/16/commits/ba52ce09e983d7f1589fe84b7ab74d13a37e005b